test/test_commands.py

```diff
@@ -159,6 +159,21 @@ def test_reimport(self):
             ['git', 'remote', 'remove', 'foo'],
             stderr=subprocess.STDOUT, cwd=cwd_vcstool)
 
+    def test_import_force_non_empty(self):
+        workdir = os.path.join(TEST_WORKSPACE, 'force-non-empty')
+        os.makedirs(os.path.join(workdir, './vcstool/not-a-git-repo'))
```

Comment:
The leading `./` seems unnecessary. Also instead of hard coding the forward slash this could be separate positional arguments: `os.path.join(workdir, 'vcstool', 'not-a-git-repo')`.

Reply:
Directories are `os.path.join()`'d in 53b4eb71ec44bd204458187de948649245ae1bd1